server/data/data-hbase/src/test/java/org/apache/james/rrt/hbase/HBaseRecipientRewriteTableTest.java

```diff
@@ -58,18 +59,18 @@ protected AbstractRecipientRewriteTable getRecipientRewriteTable() throws Except
     }
 
     @Override
-    protected void addMapping(String user, Domain domain, String mapping, int type) throws RecipientRewriteTableException {
+    protected void addMapping(String user, Domain domain, String mapping, Type type) throws RecipientRewriteTableException {
         switch (type) {
-        case ERROR_TYPE:
+        case Error:
             virtualUserTable.addErrorMapping(user, domain, mapping);
             break;
-        case REGEX_TYPE:
+        case Regex:
             virtualUserTable.addRegexMapping(user, domain, mapping);
             break;
-        case ADDRESS_TYPE:
+        case Address:
             virtualUserTable.addAddressMapping(user, domain, mapping);
             break;
-        case ALIASDOMAIN_TYPE:
+        case Domain:
             virtualUserTable.addAliasDomainMapping(domain, Domain.of(mapping));
             break;
         default:
```

Comment:
missing asString in exception message concatenation

Reply:
Outdated.